src/Microsoft.AspNetCore.SystemWebAdapters.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NETCOREAPP3_1_OR_GREATER
+using Microsoft.AspNetCore.Authentication;
+using System.ComponentModel.DataAnnotations;
+#endif
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+public class RemoteAuthenticationOptions
+#if NETCOREAPP3_1_OR_GREATER
+    : AuthenticationSchemeOptions
+#endif
+{
+#if NETCOREAPP3_1_OR_GREATER
+    public static readonly string[] DefaultHeadersToForward = new[]
+    {
+        "Authorization"
+    };
+
+    public static readonly string[] DefaultResponseHeadersToForward = new[]
```

Comment:
Do we need to make them public? Array values could be modified here - if we need it public, can you expose it as a `IReadOnlyCollection`?

Reply:
Good point. I suppose these don't need to be public, but it seems nice-to-have to have them available (maybe users will want to take existing default response headers to forward and add a couple to make their own list). I'll make these ImmutableArrays.